exporter/loadbalancingexporter/resolver_k8s.go

```diff
@@ -87,10 +87,10 @@ func newK8sResolver(clt kubernetes.Interface,
 		timeout = defaultListWatchTimeout
 	}
 
-	nAddr := strings.SplitN(service, ".", 2)
-	name, namespace := nAddr[0], "default"
-	if len(nAddr) > 1 {
-		namespace = nAddr[1]
+	parts := strings.Split(service, ".")
```

Comment:
Thanks for the fix, according to the changes you made, the loadbalancing exporter should work correctly (even without your fix), when using the standard `<svcName>.<namespace>` domain name format of the headless service. I am still experiencing the same issue even when using the `collector-backend.default` in the [config mentioned in the issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/44472#issuecomment-3574901170). Am I missing something here? Thanks!